tests/integration/utils/getId.js

```diff
@@ -0,0 +1,17 @@
+const data = require("../config/config");
```

Comment:
We should not create a separate file for each function. Can you move this to tests/integration/utils.js